support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -374,11 +373,16 @@ export function ThreeTierLanding({
 	const selectedContributionRatePlan =
 		contributionType === 'ANNUAL' ? 'Annual' : 'Monthly';
 
-	const productCatalogDescription = ['v1', 'v2'].includes(
+	const inResetBenefits = abParticipations.benefitsReset; // all variants
+	const inNewsPaperArchiveBenefit = ['v1', 'v2'].includes(
```

Comment:
Can we leave out the newspaper archive variation from this setup? Or are we running those concurrently?

Reply:
I did not want to disable or modify the ability of newspaper archve ab-test to run in this PR. With that in mind, there needs to ba a call that examined both abtests and constructed the benefits accordingly. I could disable in a follow up  PR if we feel the need however?